.circleci/config.yml

```diff
@@ -0,0 +1,43 @@
+version: 2
+jobs:
+  build:
+    docker:
+      - image: debian:latest
+    steps:
+      - run:
+          name: Install required packages
+          command: |
+            apt-get update -y
+            apt-get install -y curl git make python-setuptools tar
+      - run:
+          name: Install Arduino platform
+          command: |
+            cd /tmp
+            curl --silent https://downloads.arduino.cc/arduino-1.8.5-linux64.tar.xz | tar --xz --extract
+            mv arduino-1.8.5 /usr/share/arduino
+      - run:
+          name: Install Arturo build tool
+          command: |
+            cd /tmp
+            git clone https://github.com/sourcebots/Arturo
```

Comment:
Could we clone to `/tmp` and use `make -C` (or whatever it is) rather than keep `cd`-ing?

Reply:
Yep, 7ade5ed.